cmd/ftl/cmd_schema_import.go

```diff
@@ -136,47 +135,31 @@ func query(ctx context.Context, prompt string) error {
 func (s *schemaImportCmd) setup(ctx context.Context) error {
 	logger := log.FromContext(ctx)
 
-	nameFlag := fmt.Sprintf("name=^/%s$", ollamaContainerName)
-	output, err := exec.Capture(ctx, ".", "docker", "ps", "-a", "--filter", nameFlag, "--format", "{{.Names}}")
-	if err != nil {
-		logger.Errorf(err, "%s", output)
-		return err
-	}
-
-	port := strconv.Itoa(s.OllamaPort)
-
-	if len(output) == 0 {
+	exists, err := container.DoesExist(ctx, ollamaContainerName)
+	if !exists {
 		logger.Debugf("Creating docker container '%s' for ollama", ollamaContainerName)
 
 		// check if port s.OllamaPort is already in use
 		l, err := net.Listen("tcp", fmt.Sprintf("127.0.0.1:%d", s.OllamaPort))
-		if err == nil {
```

Comment:
WAT!

Reply:
I suspect it was an unnoticed bug! Before I did the code change it complained the port was already used, even though nothing was listening.